packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -265,7 +265,8 @@ const ruleset: any = {
       severity: "error",
       resolved: true,
       formats: [oas2],
-      given: "$[paths,'x-ms-paths'][?(!@property.endsWith('}') && !@property.endsWith('operations'))][get].responses.200.schema.properties",
+      given:
+        "$[paths,'x-ms-paths'][?(!@property.endsWith('}') && !@property.endsWith('operations') && !@property.endsWith('default'))][get].responses.200.schema.properties",
```

Comment:
>!@property.endsWith('default') [](http://example.com/codeflow?start=98&length=30)

we discussed that this rule needs to work for paths ending with other static strings as well not just default. We also discussed that the way to apply the rule could be by counting the number of segments after the providerNamespace section of the api path. The rule should be applied when the GET is against odd number of segments after the namespace segments and not apply for GETs against even number of segments.

Reply:
Addressed all the static strings.